ResearchKit/Common/ORKStepHeaderView.m

```diff
@@ -114,10 +114,15 @@ - (instancetype)initWithFrame:(CGRect)frame {
             [self addSubview:_instructionLabel];
         }
         
+        [_captionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+        [_instructionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
```

Comment:
Are these necessary at here? I assume `updateConstraintConstants` will always be called.